### PR TITLE
:feat am creat un buton si am atasat un text #58

### PR DESCRIPTION
**Pull request by antonia2203**

### CODE REVIEW

File: src/antonia2203/button.pug [hunk 1]
```diff
@@ -0,0 +1,10 @@
+html 
+  head 
+    title Button Tag 
+  body(
+    style="text-align:center; background-color: pink"
+  )
+    h1 The button Element
```
Comment (MariaTeo) [inline src/antonia2203/button.pug]: nesting a heading element inside of a button element will cause failures.

please check this link as well:
https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/no-heading-inside-button.md
image

File: src/antonia2203/button.pug [hunk 2]
```diff
@@ -0,0 +1,10 @@
+html 
+  head 
+    title Button Tag 
+  body(
+    style="text-align:center; background-color: pink"
+  )
+    h1 The button Element 
+    button(type="button", onclick='alert(\'Hello world!\')') Click Me!
```
Comment (MariaTeo) [inline src/antonia2203/button.pug]: button shouldn't be inside button.

Comment (MariaTeo) [inline src/antonia2203/button.pug]: also, this button has two attributes. format it to look pretty, on multilines

**Review by MariaTeo — CHANGES_REQUESTED** [review]

**Review by flivanov92 — APPROVED** [review]
😵

**Review by MariaTeo — APPROVED** [review]

**Pull request merged**